configs/hieraseg/deeplabv3plus_r101-d8_512x1024_80k_cityscapes_hiera_triplet.py

```diff
@@ -0,0 +1,16 @@
+_base_ = [
+    '../_base_/models/deeplabv3plus_r50-d8_vd_contrast.py',
+    '../_base_/datasets/cityscapes.py', '../_base_/default_runtime.py',
+    '../_base_/schedules/schedule_80k.py'
+]
+model = dict(
+    pretrained='https://assets-1257038460.cos.ap-beijing.\
+        myqcloud.com/resnet101_v1d.pth',
```

Comment:
Can the pre-trained weight be loaded correctly?

Reply:
`pretrained` has been set `None`, and original code has been annotation.